server/go.mod

```diff
@@ -6,8 +6,12 @@ require (
 	github.com/coreos/go-semver v0.3.0
 	github.com/coreos/go-systemd/v22 v22.3.2
 	github.com/dustin/go-humanize v1.0.0
-	github.com/form3tech-oss/jwt-go v3.2.3+incompatible
+<<<<<<< HEAD
```

Comment:
There seems to still be an issue with some rebase leftover.

Reply:
Yeah, accepted the incoming changes. The issue was on this file, let me commit and then squash them.